_data/projects/ombi.yml

```diff
@@ -0,0 +1,12 @@
+name: "Ombi"
+desc: >
+  Want a Movie or TV Show on Plex or Emby? Use Ombi! 
+  Ombi is a user friendly interface for family and friends to request new content on your media server.
+site: https://github.com/tidusjar/ombi
+tags:
+- ".NET"
+- "TypeScript"
+- "Angular"
+upforgrabs:
+  name: help wanted
+  link: https://github.com/tidusjar/ombi/labels/help wanted
```

Comment:
I _think_ this needs to be URL-encoded to address the current CI failure: `https://github.com/tidusjar/ombi/labels/help%20wanted`